Makefile

```diff
@@ -15,6 +15,8 @@ GO_GCFLAGS ?= -gcflags=all='-N -l'
 GO=GO111MODULE=on GOFLAGS=-mod=vendor go
 GO_BUILD_RECIPE=CGO_ENABLED=0 $(GO) build $(GO_GCFLAGS)
 
+OUT_DIR ?= bin
```

Comment:
This change is to make my local dev workflow more flexible, could be extracted into a new PR